brjs-sdk/sdk/libs/javascript/br-presenter/src/br/presenter/control/selectionfield/JQueryAutoCompleteControl.js

```diff
@@ -163,4 +161,12 @@ JQueryAutoCompleteControl.prototype.setOptions = function(mOptions) {
 	}
 };
 
+/**
+ * Destroy created listeners and jQuery autocomplete plugin
+ */
+JQueryAutoCompleteControl.prototype.destroy = function() {
+	this.m_oPresentationNode.value.removeListener(this._valueChangedListener);
```

Comment:
@michalvankodev can we add a gueard here in case detroy is called without any nodes being set... e.g. if(this._valueChangedListener). similarly a guard should be added for this.m_eElement on the line below